internal/cmd/providers.go

```diff
@@ -151,6 +154,7 @@ $ infra providers add okta --url example.okta.com --client-id 0oa3sz06o6do0muoW5
 	cmd.Flags().StringVar(&opts.URL, "url", "", "Base URL of the domain of the OIDC identity provider (eg. acme.okta.com)")
 	cmd.Flags().StringVar(&opts.ClientID, "client-id", "", "OIDC client ID")
 	cmd.Flags().StringVar(&opts.ClientSecret, "client-secret", "", "OIDC client secret")
+	cmd.Flags().StringVar(&opts.Kind, "kind", "", "The identity provider kind. One of 'oidc, okta, azure, or google'")
```

Comment:
Should there be a default?

Reply:
It defaults to 'oidc' on the server-side, but I have set the default now here too for understand-ability.